### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by dmc1899**

### CODE REVIEW

File: find_db_usage_in_repos [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+##                     ./input/tables/<table_name>.txt
+##                     ./input/repositories.txt
+##                     ./output/<repo_name>.txt
+##
+##   To override defaults, issue the following command:
+##    ./find_db_usage_in_repos <DB_TABLE_NAMES_FILE_PATH> \
+##                              <REPOSITORY_NAMES_FILE_PATH> \
+##                              <DB_TABLE_COLUM_NAMES_DIR_PATH> \
+##                              <RATE_LIMIT_SLEEP_INTERVAL_SEC> \
+##                              <RATE_LIMIT_SLEEP_INTERVAL_SEC> \
```
Comment (akomisarek) [inline find_db_usage_in_repos]:
```suggestion

```
Duplicated RATE_LIMIT_SLEEP_INTERVAL_SEC parameter doc

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+4. Name of the file containing the Table and Column name
+5. Path of the file in the target repository.
+
+```
+customer,id_customer,acme-organisation/repository1,my_big_query.sql,materialized_views/subfolder1/subfolder2/my_big_query.sql
+```
+The output files can then be imported into Excel for further analyses.
+
+### Limitations
+The script basically performs a primitive search for the name of the table AND the name of each column in a single search operation.
```
Comment (akomisarek) [inline README.md]: Maybe it would be worth mentioning rate limiting in limitations as well.

**Review by akomisarek — COMMENTED** [review]
Thanks, Darragh, for sharing this. I have tested my use cases and was getting the expected results + some false positives, as mentioned, but definitely a good start for doing manual investigations. 

Left some small comments!

File: find_db_usage_in_repos (hunk 1 above)
Comment (dmc1899) [reply find_db_usage_in_repos]: Removed as requested, thanks.

File: README.md (hunk 2 above)
Comment (dmc1899) [reply README.md]: Added second limitation as requested.

**Pull request merged**